Spigot-API-Patches/0267-Allow-adding-items-to-BlockDropItemEvent.patch

```diff
@@ -0,0 +1,19 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: BillyGalbreath <blake.galbreath@gmail.com>
+Date: Wed, 20 Jan 2021 14:25:26 -0600
+Subject: [PATCH] Allow adding items to BlockDropItemEvent
+
+
+diff --git a/src/main/java/org/bukkit/event/block/BlockDropItemEvent.java b/src/main/java/org/bukkit/event/block/BlockDropItemEvent.java
+index a0f6f1af304190b4c5db4b284d460f625eeb7801..3dd4bd38e72c04e74e5787fb38ca9abd10bad06b 100644
+--- a/src/main/java/org/bukkit/event/block/BlockDropItemEvent.java
++++ b/src/main/java/org/bukkit/event/block/BlockDropItemEvent.java
+@@ -64,7 +64,7 @@ public class BlockDropItemEvent extends BlockEvent implements Cancellable {
+      * Gets list of the Item drops caused by the block break.
+      *
+      * This list is mutable - removing an item from it will cause it to not
+-     * drop. It is not legal however to add new items to the list.
++     * drop. Adding to the list is allowed.
```

Comment:
do we do paper comments in javadocs? Not sure, prolly fine, git blame to the rescue.

Reply:
Yes: 
```java
/**
  * cool
  * also mini is cooler <!-- Paper -->
  */
```